superset-frontend/src/components/Select/Select.tsx

```diff
@@ -361,13 +376,20 @@ const Select = ({
   };
 
   useEffect(() => {
-    const foundOption = hasOption(searchedValue, selectOptions);
```

Comment:
Are you sure that we should remove this one? As far as I remember, this should check whether the option exists to avoid unnecessary requests against the backend.

Reply:
Yes. The problem with this check is that a search can result in more than one page, which means that we can have results that were not fetched yet and we should fire the request.